### PR TITLE
More disposable shitposter domains

### PR DESCRIPTION
**Pull request by joshtronic**

### CODE REVIEW

File: disposable_email_blacklist.conf [hunk 1]
```diff
@@ -79,10 +79,12 @@ a-bc.net
 a.mailcker.com
 a.vztc.com
 a45.in
+aaathats3as.com
 abyssmail.com
 afrobacon.com
 ag.us.to
 agedmail.com
+airmail.cc
```
Comment (martenson) [inline disposable_email_blacklist.conf]: from their website:

> Airmail is an invite-only E-mail server focused toward professionals and E-mail enthusiasts. To report abuse please E-mail abuse@airmail.cc.

**Comment by martenson** [comment]
Please remove airmail.cc and send them an email about spam abuse you faced. The rest looks valid.

Thank you.

File: disposable_email_blacklist.conf (hunk 1 above)
Comment (joshtronic) [inline disposable_email_blacklist.conf]: Found the domain here -- https://cock.li/ -- Haven't had any good traffic from any of their domains, figured I'd include it

**Comment by martenson** [comment]
Thank you very much @joshtronic !

**Comment by joshtronic** [comment]
:+1:

**Pull request merged**